ams/dashboard/src/views/catalogs/Detail.vue

```diff
@@ -45,27 +45,46 @@
             <span v-else>{{formState.catalog.optimizerGroup}}</span>
           </a-form-item>
           <a-form-item>
-            <p class="header">{{$t('storageConfig')}}</p>
+            <p class="header">{{$t('storageConfigName')}}</p>
           </a-form-item>
-          <a-form-item
-            v-for="config in formState.storageConfigArray"
-            :key="config.label"
-            :label="config.label"
-            class="g-flex-ac">
-            <a-upload
+          <a-form-item label="Type" :name="['storageConfig', 'storage.type']" :rules="[{ required: isEdit }]">
```

Comment:
@minteliuwm Can you help review the changes in Detail.vue? Some changes were made to Storage Type, Authentication Type, and Table Properties.😊

<img width="1194" alt="image" src="https://github.com/NetEase/amoro/assets/103108928/c6b30ebe-0690-43cd-9afd-578fbdd3acb9">